src/baserow/types.py

```diff
@@ -62,7 +66,8 @@ class Application:
   name: str
   order: int
   type: str
-  group: Group
+  group: t.Optional[Group]
```

Comment:
This needs a default value if the `"group"` key is not present in the payload being deserialized. Because of how dataclasses work, that means the line needs to move to the end of the class definition.

```suggestion
  group: t.Optional[Group] = None
```

Reply:
ok, done